resources/cosign-sign-attest.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# cosign-sign-attest
+source $SCRIPTDIR/common.sh
+
+base64d() {
+  base64 -d <<< "$1"
+}
+
+function full-image-ref() {
+  local url=$(cat $BASE_RESULTS/buildah-rhtap/IMAGE_URL)
+  local digest=$(cat $BASE_RESULTS/buildah-rhtap/IMAGE_DIGEST)
+  echo "$url@$digest"
+}
+
+function cosign-login() {
+  # Fixme: It would be better not to hard code quay.io here but we're
+  # already doing that in other places, e.g. the buildah login
+  cosign login -u "$QUAY_IO_CREDS_USR" -p "$QUAY_IO_CREDS_PSW" quay.io
```

Comment:
If you really want to do this, could do something like, though the environment variables are a bit indicative here:

```suggestion
  local ref="$(full-image-ref)"
  cosign login -u "$QUAY_IO_CREDS_USR" -p "$QUAY_IO_CREDS_PSW" "${ref/\/*}"
```

Reply:
Nice idea, thanks.

Reply:
Done in upcoming revision.